src/main/java/uk/co/ramp/covid/simulation/population/Population.java

```diff
@@ -334,11 +298,8 @@ public void timeStep(Time t, DailyStats dStats) {
     }
     
     public void timeStep(Time t, DailyStats dStats, ContactsWriter contactsWriter) {
-        // Movement places people in "next" buffers (to avoid people moving twice in an hour)
-        for (Household h : households) {
+        for (Household h : places.getHouseholds()) {
             h.handleSymptomaticCases(t);
-            h.doInfect(t, dStats, contactsWriter);
-            h.determineMovement(t, dStats, getPlaces());
         }
 
         for (Place p : places.getAllPlaces()) {
```

Comment:
Is the order of the list of places important here, since the first step in determineMovement() is usually checking if people have already moved?

I have a vague recollection (which is probably wrong) that doing household movement before other movement was a deliberate choice.  I think the household movement is still happening first, but the behaviour seems a bit hidden.  (I don't know what to do about this, if anything.)

Reply:
The ordering of movement shouldn't matter anymore (It quite possibly did for a bit!): it's the main reason we have the two step - decide where to go, then commit it system. Most of the checking if they have already moved is in case a family member has forced you to leave etc (As an aside while I think about this a bit: this family leave ordering does still matter - and I need to get a better story for it since it's currently best effort if everyone who goes together leaves together)